### ISSUE TITLE
Override default sprite size (or other options) via text config file.

### ISSUE THREAD

**Issue by oliverhermanni** [YoYoGames/GameMaker-Bugs#8437]
### Is your feature request related to a problem?

Whenever I start a new project and use different sprites I need to reset the default sprite dimensions in the preferences. As I use different assets in different projects this is a bit annoying. 

### Describe the solution you'd like

There could be a text config file that overrides any option like Jetbrains does in its IDEs with ".editorconfig". Settings for sprite dimensions then could look something like:

```
[...]
default_animation_speed=60
default_sprite_width=8
default_sprite_height=8
[...]
```

If a project is loaded in Gamemaker it checks for the config file and if it exists the default settings from Preferences will be overridden. 

A text file with the options set in "Preferences" could be generated via a button on the "Preferences" start page.

### Describe alternatives you've considered

Add the sprite dimensions to "Game Options". On a new project they default to the settings in "Preferences", but if changed the sprite editor uses the settings from "Game Options".

TBH this was the first idea I had, but while writing this ticket I found the other option better.

**Comment by YYDan** [comment]
Yeah, I read the initial suggestion and wondered "why wouldn't we just add it to Game Options?", save making an entirely new and obscure system for this ;)
